api-testing/multicloud-api/.gitignore

```diff
@@ -0,0 +1,11 @@
+*.iml
```

Comment:
Any specific reasons for this file?

Reply:
No need this file during build process its IntelliJ's own project structure.

Reply:
If not required, please remove

Reply:
@himanshuvar I have removed. Please review.